erdpy-up.py

```diff
@@ -155,6 +155,7 @@ def ensure_folder(folder):
 def install_erdpy():
     logger.info("Installing erdpy in virtual environment...")
     erpy_versioned = "erdpy" if not exact_version else f"erdpy=={exact_version}"
+    run_in_venv(["python3", "-m", "pip", "install", "--upgrade", "pip"])
```

Comment:
Do you think you can use the convention with checking the `return_code`, as below?

```
return_code = run_in_venv(["python3", "-m", "pip", "install", "--upgrade", "pip"])
if return_code != 0:
        raise InstallError("Could not upgrade pip.")
```